documentation/book/ref-kafka-bridge-producer-configuration.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-bridge-configuration.adoc
+
+[id='ref-kafka-bridge-producer-configuration-{context}']
+= Kafka Bridge Producer configuration
+
+Kafka Bridge producer is configured using the properties in `KafkaBridge.spec.producer`.
+This property contains the Kafka Bridge producer configuration options as keys.
+The values can be one of the following JSON types:
+
+* String
+* Number
+* Boolean
+
+Users can specify and configure the options listed in the {ApacheKafkaProducerConfig} with the exception of those options which are managed directly by {ProductName}.
+Specifically, all configuration options with keys equal to or starting with one of the following strings are forbidden:
+
+* `ssl.`
+* `sasl.`
+* `security.`
+* `bootstrap.servers`
+* `group.id`
```

Comment:
Does this apply for producer? Or only for consumer?

Reply:
We have the same list in ref-kafka-bridge-consumer-configuration.adoc. Okay?

Reply:
For consumer this should be ok. But for producer, the `group.id` IMHO should not be there.